modules/gis/src/lib/flatGeojson-to-binary-types.ts

```diff
@@ -0,0 +1,46 @@
+import type {TypedArray} from '@loaders.gl/schema';
```

Comment:
To stay in line with the usual loaders naming style, I'd suggest the file name be `flat-geojson-to-binary-types.ts` (applies to some other files as well)

Reply:
Done